src/ducks/Signals/signalFormManager/signalCrudForm/formParts/TriggerProjectsSelectorWrapper.js

```diff
@@ -0,0 +1,103 @@
+import React from 'react'
+import { graphql } from 'react-apollo'
+import { compose } from 'recompose'
```

Comment:
Compose from redux. Better.

Reply:
A lot of components import from recompose
![image](https://user-images.githubusercontent.com/14061779/60660847-52d1ac80-9e61-11e9-97ae-2b7f671cd17a.png)
Will change